src/components/Footer/Footer.module.scss

```diff
@@ -0,0 +1,16 @@
+.footer {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background-color: #ffe9e0;
+  color: #230903;
+  height: 50px;
+  width: 100vw;
+  padding: 5px 10px;
```

Comment:
This order actually seems reasonable. You have flex stuff together, then some colors, then fonts. So even you might not be aware you have done that. Kudos for that.

I would recommend an outside in approach -> https://webdesign.tutsplus.com/outside-in-ordering-css-properties-by-importance--cms-21685a. So the CSS does not go completely random and it is easier to spot attributes and overall helps with the readability